master/static/srv/check_idle.py

```diff
@@ -32,14 +33,18 @@ def main():
         port = str(os.environ["NOTEBOOK_PORT"])
         notebook_id = str(os.environ["DET_TASK_ID"])
         notebook_server = f"http://127.0.0.1:{port}/proxy/{notebook_id}"
+        master_url = str(os.environ["DET_MASTER"])
+        cert = certs.default_load(master_url)
+
         try:
             api.put(
-                str(os.environ["DET_MASTER_HOST"]),
+                master_url,
                 f"/api/v1/notebooks/{notebook_id}/report_idle",
                 {
                     "notebook_id": notebook_id,
                     "idle": get_execution_state(notebook_server),
                 },
+                cert=cert,
             )
         except Exception as e:
             print(e)
```

Comment:
let's make this log a warning with the whole exception, rather than just the error string.  It's too ambiguous like this.